cmd/utils/flags.go

```diff
@@ -166,13 +166,23 @@ var (
 		Usage: "Enable light client mode (replaced by --syncmode)",
 	}
 	ULCModeConfigFlag = cli.StringFlag{
-		Name:  "ulcconfig",
+		Name:  "les.ulcconfig",
 		Usage: "Config file to use for ULC mode",
 	}
 	OnlyAnnounceModeFlag = cli.BoolFlag{
-		Name:  "onlyannounce",
+		Name:  "les.onlyannounce",
 		Usage: "Les server sends only announce",
```

Comment:
Does not belong to this PR but always get troubles with "Les" instead of "LES". It is an acronym like "ULC. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms.